pages/destinations/[id].js

```diff
@@ -0,0 +1,94 @@
+import Head from "next/head";
+import styled from "styled-components";
+import { useRouter } from "next/router";
+import useLocalStorage from "../../hooks/useLocalStorage";
+import { dummyTrips, dummyDestinations } from "../../db";
+import DestinationItem from "../../components/DestinationItem";
+import arrowBack from "../../assets/arrowBack.svg";
+import Link from "next/link";
+import Image from "next/image";
+
+export default function Destinations() {
+  const router = useRouter();
+  const { id } = router.query;
+  const [trips, setTrips] = useLocalStorage("trips", dummyTrips);
+  const [destinations, setDestinations] = useLocalStorage(
+    "destinations",
+    dummyDestinations
+  );
+
+  let countryName = trips.find((trip) => trip.id == id)?.country || "Not found";
```

Comment:
You should use the strict equality operator trip.id === id (works when your ids are strings ;) )

Reply:
Done - improved the dummy data to use the proper strict equality operator.